core/templates/components/state-editor/state-interaction-editor/state-interaction-editor.directive.ts

```diff
@@ -112,11 +112,17 @@ angular.module('oppia').directive('stateInteractionEditor', [
           $scope.adjustPageHeight = function() {};
 
           $scope.getCurrentInteractionName = function() {
-            return (
-              StateInteractionIdService.savedMemento ?
+            return {
+              header: StateInteractionIdService.savedMemento ?
+              '(' +
+              INTERACTION_SPECS[StateInteractionIdService.savedMemento].name +
+            ')' : '',
+              body: StateInteractionIdService.savedMemento ?
                 INTERACTION_SPECS[StateInteractionIdService.savedMemento].name :
-                '');
+                '',
+            };
```

Comment:
Maybe we can just do this in the template? :)

Reply:
DONE